agent/engine/docker_task_engine_test.go

```diff
@@ -330,40 +315,39 @@ func TestRemoveEvents(t *testing.T) {
 			ExitCode: &exitCode,
 		},
 	}
+	steadyStateVerify <- time.Now()
```

Comment:
Should we move this line before sending the stop event, there may be a race condition where `waitEvent` is waiting for both the `dockerMessages` and `steadStateVerify` channel?